examples/bq_file_load_benchmark/benchmark_tools/staging_table_generator.py

```diff
@@ -0,0 +1,221 @@
+                # created in create_resized_table() as the source base table.
+                # Otherwise, if the file size of the current iteration is
+                # greater than the last iteration, use the resized table from
+                # the previous iteration as the source base table when running
+                # so that each table can take advantage of the size of the last.
+                size = sizes[i]
+                if size > sizes[i - 1]:
+                    base = sizes[i - 1]
+                    source_table = get_resized_table_name(
+                        table.table_id,
+                        base
+                    )
+                    source_dataset = self.resized_dataset_id
+                else:
+                    source_table = table.table_id
+                    source_dataset = self.staging_dataset_id
+
+                destination_table = get_resized_table_name(
+                    table.table_id,
+                    size
+                )
+                target_gb = size
+                command_str = ('python {0:s}/bq_table_resizer.py '
+                               '--project {1:s} '
+                               '--source_dataset {2:s} '
+                               '--source_table {3:s} '
+                               '--destination_dataset {4:s} '
+                               '--destination_table {5:s} '
+                               '--target_gb {6:f} ')
+                command = command_str.format(
+                    bq_resizer_path,
+                    self.project,
+                    source_dataset,
+                    source_table,
+                    self.resized_dataset_id,
+                    destination_table,
+                    target_gb,
+                )
+                os.system(command)
+                logging.info('Created resized table from {0:s}'.format(
+                    source_table
+                ))
+                logging.info('Resized table complete: {0:s}'.format(
+                    destination_table
+                ))
+
+
+def get_resized_table_name(table_id, size):
+    """Creates a name for resized tables.
+
+    Ensures that the file size in the resized staging table name is greater
+    than 1, since BiqQuery tables names can not have '.' characters.
+
+    Args:
+        table_id(str): Name of the staging table used to generate the
+            resized staging table.
+        size(float): Size of the resized staging table in GB.
+
+    Returns:
+        The name of the resized staging table containing a size and unit
+        without decimals in a string format. For example, if the provided
+        table_id was 100_STRING_10 and the provided size was .01, the returned
+        name would be 100_STRING_10_10MB.
+    """
+    if size >= 1:
+        label_size = size
+        label_unit = 'GB'
+    elif size >= .001:
+        label_size = int(size * 1000)
```

Comment:
Avoid "magic numbers" in favor of assigning to a descriptive var name. 
For Example `BYTES_IN_MB` or similar.